### DISCUSSION TITLE
FR: Section/source groups

### DISCUSSION THREAD

**Discussion by thupsi** [craftcms/cms#14609] [Ideas] [answered]
Since the announcement of the entrification plan and the delay/reconsidering of the Unified Content View we are left in a in-between state where all elements are lumped together under the "Entries" view. It's not the end of the world - we are not forced to entrify all the things yet. It does seem though that, currently, we have to sacrifice some UX if we want to go down the entrification path.

I have a simple (conceptually at least) suggestion that would provide parity of content organization between pre and post entrification states, right now: Section groups.

"Section groups" would be groups of sections/sources (suprise!) designated by the admin/developer, that would live in their own place in the global sidebar, like "Categories" and "Globals" do now. At least from a UI point of view, no major work would be needed. Even the recent "icon" field would serve to provide the icon of each section group in the sidebar.

I realise P&T may not want to make rush changes. It's important that they approach content presentation in a holistic and considered way - it's the heart of any CMS! So if it takes two years for Craft 6, so we can have a great solution, so be it. But, I also think my simple suggestion is worth considering because it's not introducing a new UI/UX paradigm, it's not forcing a new direction, and thus it leaves the team to continue evolving things in any way they choose down the road.

**Comment by brandonkelly** [comment]
We’re considering something like this, as an alternative to doing the unified content editor.

**Reply by thupsi** [reply]
If you do this and also ungroup single sections (something like the "expanded singles" plugin), I think you will cover a lot of bases. And those two improvements demand basically no new UI design work.

**Comment by brandonkelly** [comment, marked as answer]
Craft 5.9.0 is out with the ability to create custom entry index pages (#17779) 🎉
